Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -296,13 +291,13 @@ void Renderer::DrawDebugText()
   }
 
   // OSD Menu messages
-  if (OSDChoice > 0)
+  if (m_osd_message > 0)
   {
-    OSDTime = Common::Timer::GetTimeMs() + 3000;
-    OSDChoice = -OSDChoice;
+    m_osd_time = Common::Timer::GetTimeMs() + 3000;
+    m_osd_message = -m_osd_message;
   }
 
-  if ((u32)OSDTime > Common::Timer::GetTimeMs())
+  if ((u32)m_osd_time > Common::Timer::GetTimeMs())
```

Comment:
`static_cast` while you're at it?

Reply:
Sure, why not. (Although I'm not willing to go any further because the surrounding code is also kinda messy tbh).